### PR TITLE
Fix 'recommended modules' popin behavior in BO

### LINKED ISSUE #9593: [BOOM-5953] Install a recommanded module in a symfony page
> This issue has been migrated from the Forge. Read the [original ticket here](http://forge.prestashop.com/browse/BOOM-5953).

- _**Reporter:**_ marion_francois
- _**Created at:**_ Wed, 11 Jul 2018 15:20:57 +0200

<p>MBO module is uninstalled</p>

<p>Go on Shipping > carriers<br/>
 Now, click on "recommanded modules" in the header of the carriers' page<br/>
 Install a recommanded module like mondial relay<br/>
 You have this: </p>

<p><span class="image-wrap" style=""><a id="69371_thumb" href="http://forge.prestashop.com/secure/attachment/69371/69371_image-2018-07-11-15-19-54-765.png" title="image-2018-07-11-15-19-54-765.png" file-preview-type="image" file-preview-id="69371" file-preview-title="image-2018-07-11-15-19-54-765.png"></a></span></p>

<p> And the module isn't installed</p>

<p>If I try with Boxtal module, it's working</p>

<p> </p>
        
* How to reproduce the issue ?

N/A

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix the 'recommended modules' popin behavior
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixed https://github.com/PrestaShop/PrestaShop/issues/9593
| How to test?  | Go on 'carriers' page in BackOffice `/admin-dev/index.php?controller=AdminCarriers`/ and click on 'recommended modules' button in the top right corner of the screen. Install one module, uninstall another. The display is not pretty, but it works.

This is a duplicate of https://github.com/PrestaShop/PrestaShop/pull/9317 but for `develop`.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9352)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Module/tab-modules-list.html.twig [hunk 1]
```diff
@@ -23,7 +23,9 @@
  * International Registered Trademark & Property of PrestaShop SA
  *#}
 {% if modulesList is defined and modulesList is not empty %}
-<div class="row row-margin-bottom">
+<script src="{{ asset('themes/new-theme/public/module_card.bundle.js') }}"></script>
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Resources/views/Admin/Module/tab-modules-list.html.twig]: Script at the end of the file please :)

**Comment by matks** [comment]
Requested changes applied, commit amended and branch rebased on develop

**Comment by matks** [comment]
Branch rebased on `develop` again as `develop` has evolved quite a lot in 15 days 😛 

Please merge before it needs to be rebased again ^^ @eternoendless @PierreRambaud @tomlev ?

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by mickaelandrieu** [comment]
Hello @matks,

sorry for that, you can use @Quetzacoalt91 "trick" for assets: you commit them only when someone add "QA-approved" label. Our QA team knows how to update assets, you just need to tell them in pull request description.

In the next major, we will *probably* remove them from the Core, to ease the contribution for everyone.

**Comment by mickaelandrieu** [comment]
Hello @matks, one rebase / cs fix and I merge ! 

Good job :)

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thanks @matks and everyone for review

**Pull request merged**